ride_share.rb

```diff
@@ -0,0 +1,76 @@
+ride_share = [
+    DR0001 = [{driver_id: "DR0001", date: "3rd Feb 2016", cost: 10, rider_id: "RD0003", rating: 3 }, 
+             {driver_id: "DR0001", date: "3rd Feb 2016", cost: 30, rider_id: "RD0015", rating: 4 },
+            {driver_id: "DR0001", date: "5th Feb 2016", cost: 45, rider_id: "RD0003", rating: 2 }
```

Comment:
This syntax is probably not doing what you expect. When you say `DR001 = ...`, you're creating a constant named `DR001`, in addition to adding a value to the `ride_share` array. You could omit the `DR001 = ` and your code would work the same way.